doc/users.rst

```diff
@@ -1,602 +1,303 @@
 .. _users:
 
 User Guide
-===========
+============
 
-Getting Started
----------------
-
-To get started with foreshadow, install the package using pip install. This will also
-install the dependencies. Now create a simple python script that uses all the
-defaults with Foreshadow.
+Introduction
+============
 
-First import foreshadow
+In this tutorial, we will go through a typical ML workflow with Foreshadow using a subset of the `adult data set <https://archive.ics.uci.edu/ml/datasets/Adult>`_ from the UCI machine learning repository.
```

Comment:
Minor quibble: Wonder if more common datasets out there (like Boston or IRIS used by most tutorials) that people might be more familiar with are a better fit here. [Sklearn datasets](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.datasets) are probably a good place to dig. 

Shouldn't be blocking though - adult dataset is a good representation of what you might see in real life.

Reply:
I actually spotted a bug using the Iris data set with intent override. The fix is already published to a PR. Thanks for pointing this out!